installers/build-certificates-unix.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+#
+# This is an install-time script that generates the CA/server key/cert pairs,
+# for front/backend channel security.
+# It must be run from the application install directory.
+#
+# Source: https://pki-tutorial.readthedocs.io/en/latest/simple/index.html
+#
+# WARNING: keep in sync with 'installers/build-certificates-win64.bat'
+
+echo Generating certificates.
+openssl version
```

Comment:
We have only installed XCode on our Mac-in-Cloud, so I guess it implies that, at least for our version of OS X.

Once we have Nix-driven builds we will just specify `libressl` as a build input.